src/som/interpreter/actors/Actor.java

```diff
@@ -262,12 +355,34 @@ public void uncaughtException(final Thread t, final Throwable e) {
       VmSettings.NUM_THREADS, new ActorProcessingThreadFactor(),
       new UncaughtExceptions(), true);
 
+  public static final void shutDownActorPool() {
+      actorPool.shutdown();
+      try {
+        actorPool.awaitTermination(10, TimeUnit.SECONDS);
+      } catch (InterruptedException e) {
+        // TODO Auto-generated catch block
```

Comment:
better rethrow as a unchecked `RuntimeException`.
we should not swallow any exceptions if it is not for a really good reason. Ideally that reason should be stated in a comment.